middleware/crud-functions/postWinners.js

```diff
@@ -31,6 +32,7 @@ function postWinners(userHandles) {
 
   const updated = Promise.all(promises)
     .then((_result) => {
+      cache.setWinners(userHandles);
```

Comment:
this method name doesn't match the ones on the class.

Reply:
yikes oops